src/platform/pic32/uart.cpp

```diff
@@ -83,6 +94,8 @@ void openxc::interface::uart::changeBaudRate(UartDevice* device, int baud) {
             baud) - 1);
     ((p32_uart*)_UART1_BASE_ADDRESS)->uxMode.reg = (1 << _UARTMODE_ON) |
             (1 << _UARTMODE_BRGH);
+			#warning "MG UEN"
```

Comment:
Should have taken the warning out. The subsequent line U1MODEbits.UEN = 2; was added because uxMode register is re-initialized during the call to changeBaudRate, but not all of the potentially relevant settings within the register (i.e. peripheral flow control) that the application may have already configured are preserved. 

To be fair I don't know if this was required. I made this change while searching for the reason why iOS devices were not working, thinking that it was possible that the call to begin() might be overriding an existing flow control setting that was never restored. I didn't scan through all the ways the UART driver code is called, and depending on when changeBaudRate is called (and in relation to enableFlowControl and others) it may not matter if changeBaudRate wipes out the register, but to be safe I put it in since flow control is always used.

It might be better to use the SET, INV, and CLR registers for low-level peripheral register modification, or maybe store the register before re-initializing the peripheral, then this line can come out and the call to changeBaudRate won't have any unexpected side effects.

If I've missed the mark feel free to remove and see if iOS connectivity still works.